### PR TITLE
Bugfix/tests candidates selection

### PR DESCRIPTION
**Pull request by meirhalachmi**

### CODE REVIEW

File: src/corona_hakab_model/detection_model/healthcare.py [hunk 1]
```diff
@@ -63,17 +63,10 @@ def testing_step(self):
             # Who can to be tested
             can_be_tested = self._get_testable(test_location)
             test_candidates = want_to_be_tested & can_be_tested
-            test_candidates_inds = set(np.flatnonzero(test_candidates))
+            test_candidates_inds = set(np.flatnonzero(test_candidates))  # TODO: This might be slow on large population
```
Comment (bgmoshe) [inline src/corona_hakab_model/detection_model/healthcare.py]: How about np.unique then (you can drop the flatnonzero, I think, and maybe use nonzero on the unique returned

Comment (bgmoshe) [reply src/corona_hakab_model/detection_model/healthcare.py]: Sorry, might have something wrong, why this might be slow?

Comment (bgmoshe) [reply src/corona_hakab_model/detection_model/healthcare.py]: image

Comment (meirhalachmi) [reply src/corona_hakab_model/detection_model/healthcare.py]: Removed the todo

**Review by bgmoshe — APPROVED** [review]

**Pull request merged**